wp-content/themes/csisnuclearnetwork/template-parts/newsletter-block-acf.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * The about PONI section of the home page,
+ * including information about directors and a team photo
+ *
+ * @link https://developer.wordpress.org/themes/basics/template-hierarchy/
+ *
+ * @package CSIS iLab
+ * @subpackage @package nuclearnetwork
+ * @since 1.0.0
+ */
+
+?>
+
+<?php 
+    $dailyNewsletterName = get_field('daily_newsletter_name');
+    $dailyNewsletterDesc = get_field('daily_newsletter_description');
+    $dailyNewsletterLinks = "<div class='recent-news'><a href=''>Most recent newsletter" .nuclearnetwork_get_svg('chevron-right') . "</a><a href=''>All Nuclear Policy News" .nuclearnetwork_get_svg('chevron-right') . "</a></div>";
+    $monthlyNewsletterName = get_field('monthly_newsletter_name');
+    $monthlyNewsletterDesc = get_field('monthly_newsletter_description');
+    $monthlyNewsletterLinks = "<div class='recent-news'><a href=''>Most recent newsletter" .nuclearnetwork_get_svg('chevron-right') . "</a></div>";
+
+    $constructedDailyNewsletter = "<div class='daily'><h3>" . $dailyNewsletterName . "</h3><p>" . $dailyNewsletterDesc . "</p><a class='subscribe btn btn--teal btn--newsletter' href='https://www.csis.org/subscribe#section-newsletters'>Subscribe</a>" . $dailyNewsletterLinks . "</div>";
```

Comment:
The URLs for the subscribe buttons are also ACF fields, they're just located in a different spot. Those can be accessed using the code below. You'll also want to double check that they're functional - right now there's an extra set of quotation marks that break the whole link.

```
get_field('nuclear_policy_news_link', 'option')
get_field('monthly_newsletter_link', 'option')
```

Reply:
I'm not sure how to do this for the widgets, because that's code coming from the server.